interface/src/app/gene/network/network.less

```diff
@@ -5,7 +5,7 @@ line.edge {
 circle.gene {
   stroke-width: 2px;
   stroke: #AAA;
-  fill: #FFF;
+  //fill: #FFF;
```

Comment:
I commented this line out because the color of gene circles (including the default color) now is controlled completely by d3.network.js. So this color is hard coded. I may need to figure out a better way in the future.